docs/rules/no-large-snapshots.md

```diff
@@ -110,3 +110,27 @@ In an `eslintrc` file:
   }
 ...
 ```
+
+In addition there is an option for whitelisting large snapshot files. Since
+`//eslint` comments will be removed when a `.snap` file is updated, this option
+provides a way of whitelisting large snapshots. The list of whitelistedSnapshots
+is keyed first on the absolute filepath of the snapshot file. You can then
+provide an array of strings to match the snapshot names against. If you're using
+a `.eslintrc.js` file, you can use regular expressions AND strings.
+
+In an `.eslintrc.js` file:
+
+```javascript
+...
+
+  "rules": {
+    "jest/no-large-snapshots": ["error",
+      {
+        "whitelistedSnapshots": {
+          "/path/to/file.js.snap": ["snapshot name 1", /a big snapshot \d+/]
```

Comment:
should we note somewhere in the docs people can use `path.resolve(__dirname, './relative-path')` or `require.resolve('./relative-path')` to get absolute paths that work between computers?

Reply:
done.